### ISSUE TITLE
Actionscript 3.0 Tracking: flash.accessibility.AccessibilityImplementation

### ISSUE THREAD

**Issue by danielhjacobs** [danielhjacobs/test_avm2_tracking_issues#51]
This is the tracking issue for **flash.accessibility.AccessibilityImplementation** in AVM2. We will check off each item of progress as appropriate, and fill in any relevant or missing information as we continue development of Ruffle.
# Legend

Each checkbox is independent of another. It's entirely possible for something to be tested but not exist yet, or for us to believe that it's completely functional but we haven't made enough tests to prove it.
## "Property Exists"

This means the item exists, but may not necessarily be fully implemented.
## "Has Test Coverage"

This means that we believe that we have a good test coverage of this item, regardless of if those tests pass. It's okay to have tests available but not implement the item yet.
## "Completely Functional"

This means we believe that the item is completely implemented, and no more work needs to be done towards making it functional.
# flash.accessibility.AccessibilityImplementation
### errno : uint

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### stub : Boolean

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### AccessibilityImplementation()

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### accDoDefaultAction(childID:uint):void

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### accLocation(childID:uint):*

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### accSelect(operation:uint, childID:uint):void

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accDefaultAction(childID:uint):String

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accFocus():uint

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accName(childID:uint):String

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accRole(childID:uint):uint

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accSelection():Array

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accState(childID:uint):uint

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_accValue(childID:uint):String

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_selectionActiveIndex():*

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### get_selectionAnchorIndex():*

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### getChildIDArray():Array

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional

### isLabeledBy(labelBounds:Rectangle):Boolean

* [] Property Exists

* [] Has Test Coverage

* [] Completely Functional
